blog/2022/01/22/o2-orion-monopoles.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: 'O2 Beacon 3 - Orion'
+description:
+  O2's latest flagship monopoles, Orion, are a mastermind in the mobile networking world, cramming two 5G carriers and tonnes of 4G bands into a
+  single streetworks cabinet and monopole. It's a work of art.
+path: 2022/01/07/o2-beacon-3-orion
+created_at: 2022-01-22T23:00Z
+---
+
+import { FactBox, MathBlock, Tooltip, SpectrumMap } from '@blog'
+
+<TableOfContents {...props} />
+
+<FactBox title="Note">
+
+Click on any of the images in this article to view them at their full-size glory.
+
+</FactBox>
+
+# A brief history
+
+O2 (<Tooltip text="MCC: 234, MNC: 10"><code>234/10</code></Tooltip>) is the largest network operator in the United Kingdom in terms of customers.
+Including its <Tooltip text="Mobile virtual network operators">MVNOs</Tooltip>, O2's customer base totals a whopping 36.2 million customers in a
+country of just 67 million, meaning they have a market share of 54%.
+([Source](https://news.o2.co.uk/press-release/o2-continues-customer-growth-and-profitability-as-it-supports-uk-through-pandemic))
+
+In the past, O2 has had a fairly troublesome history in terms of capacity. Many parts of the UK which used to be more rural have expanded over
+time, resulting in O2 sites which used to be more than capable of providing decent performance to customers within their zone suffering from
+lower performance as network load increases each year. This is extremely prevalent with their main deployments being 800 MHz LTE across rural and
+suburban United Kingdom.
+
+# Introduction to Orion
+
+In 2021, O2 launched its **Beacon 3** network upgrade project, including its brand new, flagship Orion monopoles.
+
+<FactBox>
+
+"Beacon" was the codename for a series of joint upgrade plans made between O2 and Vodafone to provide LTE to the whole of the UK.
+
+Beacon 3 is the first Beacon project to include installation of <Tooltip text="one network operator">unilateral</Tooltip> sites as opposed to the
+multi-operator radio access network (MORAN) sites we typically see with Cornerstone Networks.
+
+This indicates a possibly deteriorating relationship between the two operators.
+
+</FactBox>
+
+Orion monopoles manage to cram a multitude of 4G bands, along with n28 and n78 5G and their standard 2G/3G all into **one street cabinet and one
+monopole**, often replacing what used to be basic 4G/3G/2G monopoles and several street cabinets.
+
+O2 have begun favouring these new deployments because of their simplicity. A **pole swap can be done in just one day**, consisting of contractors
```

Comment:
```suggestion
O2 have begun favouring these new deployments because of their simplicity. A pole swap can be done in just one day, consisting of contractors installing the feeders in the new cabinet, and an HIAB truck arriving with the new monopole, lifting and dropping it into place, and then taking the old one away to be stripped for parts.
```

Bold doesn't make much sense here.